content/en/tracing/trace_collection/compatibility/python.md

```diff
@@ -47,76 +54,90 @@ The `ddtrace` library includes support for a number of web frameworks, including
-| [urllib3][59]     | >= 1.22           | no | https://ddtrace.readthedocs.io/en/stable/integrations.html#urllib3     |
-| [graphql-core][60]| >= 2.0            | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#graphql |
-| [pytest][14]              | >= 3.0            | no | https://ddtrace.readthedocs.io/en/stable/integrations.html#pytest  |
+| Library                           | Supported Version |  Automatic       | Library Documentation                                                    |
+| -----------------                 | ----------------- | ---------------- | ------------------------------------------------------------------------ |
+| [aiobotocore][45]                 | >= 0.2.3          | no | https://ddtrace.readthedocs.io/en/stable/integrations.html#aiobotocore |
+| [asyncio][46]                     | Fully Supported   | > Python 3.7 yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#asyncio     |
+| [Botocore][47]                    | >= 1.4.51         | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#botocore    |
+| [Boto2][48]                       | >= 2.29.0         | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#boto2       |
+| [Celery][49]                      | >= 3.1            | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#celery      |
+| [Consul][50]                      | >= 0.7            | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#consul      |
+| [dogpile.cache][69]               | >= 0.9            | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#dogpile-cache      |
+| [Futures][51]                     | Fully Supported   | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#futures     |
+| [gevent][52]                      | >= 1.0            | no | https://ddtrace.readthedocs.io/en/stable/integrations.html#gevent      |
+| [graphene][70]                    | >= 3.0            | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#graphene        |
+| [graphql-core][60]                | >= 2.0            | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#graphql |
+| [Grpc][53]                        | >= 1.8.0          | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#grpc        |
+| [httplib][54]                     | Fully Supported   | no | https://ddtrace.readthedocs.io/en/stable/integrations.html#httplib     |
+| [httpx][71]                       | >= 0.17           | no | https://ddtrace.readthedocs.io/en/stable/integrations.html#httpx     |
+| [Jinja2][55]                      | >= 2.7            | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#jinja2      |
+| [Jinja2][55] [aiohttp-jinja2][63] | >= 1.5            | yes | https://ddtrace.readthedocs.io/en/stable/integrations.html#aiohttp-jinja2      |
```

Comment:
Do we maybe want to list this as aiohttp-jinja2 towards the top? Like how we split botocore and aiobotocore?